projects/packages/waf/src/class-waf-initializer.php

```diff
@@ -111,6 +111,12 @@ public static function update_waf_after_plugin_upgrade( $upgrader, $hook_extra )
 	 */
 	public static function check_for_waf_update() {
 		if ( get_option( self::NEEDS_UPDATE_OPTION_NAME ) ) {
+			// Compatiblity patch for cases where an outdated WAF_Constants class has been
+			// autoloaded by the standalone bootstrap execution at the beginning of the current request.
+			if ( ! method_exists( Waf_Constants::class, 'define_mode' ) ) {
+				return;
```

Comment:
Actually maybe we should regenerate the bootstrap here?

Reply:
Added the generation here, will re-test with this inclusion :+1: